build.gradle

```diff
@@ -7,15 +7,15 @@ import org.gradle.api.tasks.testing.logging.TestLogEvent
 plugins {
     id 'java'
     id 'jacoco'
-    id 'checkstyle'
+    //id 'checkstyle'
```

Comment:
I think we should ensure that checkstyle passes for every PR

Reply:
Agreed. I reverted back everything except for files in `src` folder.